list-branch-pr

```diff
@@ -305,25 +306,23 @@ def send(script, pulls):
 if __name__ == "__main__":
     args = parseArgs()
 
-    cache = PickledCache(".cached_github_client_cache")
+    cache = PickledCache("../.cached_github_client_cache")
```

Comment:
It would probably better to make this configurable or use $HOME. Otherwise it might fail for local tests.

Reply:
Good point, I'll make it `$HOME` in a separate PR.

Reply:
See #814